input-plugins/input-plugin-reference/index.js

```diff
@@ -31,9 +31,10 @@ module.exports = {
       options: [],
       settings: {
         empty: false,
```

Comment:
While I'm in there, should I write validation to ensure that the item chosen is one of the options selected? Or should we write an actual story for that?